src/main/frontend/handler/editor.cljs

```diff
@@ -3425,20 +3440,26 @@
                  (collapse-block! uuid))))))))))
 
 (defn- collapse-all!
-  []
-  (let [blocks-to-collapse
-        (->> (all-blocks-with-level {:collapse? true})
-             (filter (fn [b] (collapsable? (:block/uuid b)))))]
-    (when (seq blocks-to-collapse)
-      (doseq [{:block/keys [uuid]} blocks-to-collapse]
-        (collapse-block! uuid)))))
+  ([]
+   (collapse-all! nil))
```

Comment:
use `(collapse-all! nil nil)` directly to avoid a second dispatch.

Reply:
@andelf  fixed